web/packages/teleport/src/Navigation/Section.tsx

```diff
@@ -519,6 +526,30 @@ function LicenseFooter({
   );
 }
 
+export function PoweredByTeleportLogo() {
+  const theme = useTheme();
+  const src = theme.type === 'dark' ? logoPoweredByDark : logoPoweredByLight;
+  return (
+    <StyledFooterBox
+      py={3}
+      px={4}
+      pb={4}
+      css={`
+        border: none;
+      `}
+    >
+      <Image
+        src={src}
+        width="fit-content"
```

Comment:
`fit-content` doesn't give us anything (I changed the navigation width):
<img width="453" alt="image" src="https://github.com/user-attachments/assets/95724873-a3fc-4b93-a8b2-bfafc8aaee69" />

It's better to set `maxWidth: 100%` to ensure the image never overflows its container:
<img width="453" alt="image" src="https://github.com/user-attachments/assets/c0d69404-ffdb-43ac-885e-768d79b04867" />